sci-libs/primegen/primegen-0.97.recipe

```diff
@@ -43,6 +43,11 @@ BUILD_REQUIRES="
 	haiku_devel
 	"
 
+TEST_PREREQUIRES="
+	cmd:md5sum
+	"
+BUILD_PREREQUIRES+="$TEST_PREREQUIRES"
```

Comment:
On the GCI site, in a reply to another student, PulkoMandy wrote:
> Eventually we will change the tool to allow a TEST_REQUIRES or similar, but this hasn't been done yet.

Later, in PR 1812, you and him suggested using TEST_PREREQUIRES.

@pulkomandy (and others), which one do you prefer? TEST_REQUIRES or TEST_**PRE**REQUIRES ?
Shall we make an online poll using GitHub's thumbs (up and/or down) on any of the next two comments?

Reply:
TEST_PREREQUIRES

Reply:
TEST_REQUIRES

Reply:
I'm still not sure what the distinction was created for in the first place; someone would have to investigate in HaikuPorter. Perhaps @mmlr knows?

Reply:
The distinction is relevant only when cross compiling (bootstrap). pre-requires are for running on the host, whereas requires are built for the target. In the case of tests, we can only run them on target, so the distinction is not useful.